packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -576,10 +577,6 @@ class HotRunner extends ResidentRunner {
     String reloadMessage;
     final Stopwatch vmReloadTimer = Stopwatch()..start();
     try {
-      final String entryPath = fs.path.relative(
```

Comment:
Moved as the reload path depends on whether trackWidgetCreation is set which is only known given a specific device. Right now there is no way to run with multiple devices one with and one without trackWidgetCreation but that could be possible for profile builds in the future.